src/Development/IDE/Spans/AtPoint.hs

```diff
@@ -148,7 +149,8 @@ locationsAtPoint getHieFile _ pos =
                 -- This file might no longer exists and even if it does the path will be relative
                 -- to the compilation directory which we don’t know.
                 let span = setFileName srcPath $ nameSrcSpan $ availName avail
-                pure span
+                -- We provide the name for the unhelpful span case, so that an alternative method to get location from name can be used.
+                pure (span, Just name)
```

Comment:
Could you please add a similar explanation in the docstring for `locationsAtPoint`? I think the `Maybe` result is a bit confusing given that these behave a lot like lookup functions so it's easy to think "`Nothing` = not found"